src/main/java/hw_3/Task1.java

```diff
@@ -0,0 +1,33 @@
+package hw_3;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+import static java.awt.SystemColor.text;
```

Comment:
These unused imports should be removed
![Screenshot_11](https://github.com/AntBelyi/ALevel_automation/assets/66726924/5dde3895-5337-440b-ad8f-df9b7cd594cb)

Reply:
fixed

Reply:
not